yang/frr-staticd.yang

```diff
@@ -0,0 +1,150 @@
+module frr-staticd {
+  yang-version "1.1";
+  namespace "http://frrouting.org/yang/staticd";
+
+  prefix frr-staticd;
+
+  import ietf-routing-types {
+    prefix "rt-types";
+  }
+
+  import ietf-routing {
+    prefix "rt";
+  }
+
+  import ietf-ipv4-unicast-routing {
+    prefix "v4ur";
+  }
+
+  import ietf-ipv6-unicast-routing {
+    prefix "v6ur";
```

Comment:
It's probably useful to keep the model vendor in the prefix for clarity. `ietf-v6ur`, perhaps. But this is just a nit.